module1/homework/grayscaleImages/main.cpp

```diff
@@ -1,11 +1,11 @@
 #include <array>
 #include <forward_list>
 
-// TODO: include
+#include "compression.hpp"
```

Comment:
array is included in the header,so you could remove it here.

Reply:
+1